File: algorithm/src/main/java/baekjoon/p08393/Main.java
package baekjoon.p08393;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.nextInt()));
    }

    private static int solve(int n) {
        int ret = 0;

        for (int i = 1; i <= n; i++) {
            ret += i;
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p08393/Main.kt
package baekjoon.p08393

fun main(args: Array<String>) {
    val n = readLine()!!.toInt()
    println(solve(n))
}

private fun solve(n: Int): Int {
    var ret = 0

    for (i in 1..n) {
        ret += i
    }

    return ret
}